src/chef-mover/apps/oc_chef_wm/src/oc_chef_wm_authn_ldap.erl

```diff
@@ -171,6 +171,7 @@ result_to_user_ejson(LoginAttr, _, [{eldap_entry, CN, DataIn}|_]) ->
     LookupFields = [{"displayname", <<"display_name">>},
                     {"givenname", <<"first_name">>},
                     {"sn", <<"last_name">>},
+                    {"cn", <<"common_name">>},
```

Comment:
Let's remove this one -- LDAP  auth components aren't used in chef-mover, and we want to keep the chef-mover vendored apps as frozen as we can.